src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -75,11 +76,27 @@ protected override void ProcessRecord()
             foreach (object errorRecord in errorRecords)
             {
                 PSObject obj = PSObject.AsPSObject(errorRecord);
-                obj.TypeNames.Insert(0, "PSExtendedError");
 
-                // Remove some types so they don't get rendered by those formats
-                obj.TypeNames.Remove("System.Management.Automation.ErrorRecord");
-                obj.TypeNames.Remove("System.Exception");
+                if (obj.TypeNames.Contains("System.Management.Automation.ErrorRecord"))
+                {
+                    if (!obj.TypeNames.Contains("System.Management.Automation.ErrorRecord#PSExtendedError"))
```

Comment:
it's probably ok, but since you use this string a bit, you might turn it into a const

Reply:
Yeah, I'll turn into const.